lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -499,6 +499,50 @@ def additional_processor_response_from_result(result)
         result.transaction&.additional_processor_response
       end
 
+      def credit_card_details(result)
+        if result && result.credit_card_details
+          credit_card_details = {
```

Comment:
Just a thought - it seems to me that all that is being returned from this method is the hash.

Is there any need to set a variable `credit_card_details` equal to the output of the hash? I think you get the same result without `credit_card_details = `

Reply:
Yep that is true since Im setting credit_card_details =  in the transaction hash to equal what is returned from the helper methods! Thanks for this callout!